CRM/Core/Form.php

```diff
@@ -761,6 +761,7 @@ public function getTitle() {
    */
   public function setTitle($title) {
     $this->_title = $title;
+    $this->assign('title', $title);
```

Comment:
I'm concerned that this change may have unintended consequences. Grep turns up 65 occurrences of `$title` in the templates directory. This will likely cause some of those variables to be overwritten.

Reply:
@mattwire this seems like something that was eventually removed from here too https://github.com/civicrm/civicrm-core/pull/13781

My grepping and my testing suggest the 2 instances of ```$this->assign('title', $title); ``` below can be removed with no ill-effect

Reply:
@eileenmcnaughton Agreed - removed thankyou